spec/lib/basic_encoding_spec.rb

```diff
@@ -19,6 +19,10 @@ def bytes_to_binary_string( bytes, claimed_encoding = nil )
 windows_1258_string = bytes_to_binary_string [ 0x44, 0x4f, 0x4e, 0x47, 0x20,
                                                0xFE, 0x20, 0x44, 0x4f, 0x4e,
                                                0x47 ]
+# 0x92 is invalid
```

Comment:
Invalid in any encoding/charset etc, or just invalid UTF8? Might want to be a bit clearer on this comment since this is a very infrequently looked at area of the code!

Reply:
Updated the comment